generator/internal/language/templates/rust/crate/src/transport.rs.mustache

```diff
@@ -56,11 +56,11 @@ impl crate::traits::{{NameToPascal}} for {{NameToPascal}} {
     ) -> Result<{{OutputTypeName}}> {
         let options = options.set_default_idempotency(reqwest::Method::{{HTTPMethod}}.is_idempotent());
         let builder = self.inner.builder(
-            reqwest::Method::{{HTTPMethod}}, format!("{{HTTPPathFmt}}"
-               {{#HTTPPathArgs}}
-               , {{{.}}}
-               {{/HTTPPathArgs}}
-            ))
+            reqwest::Method::{{HTTPMethod}},{{#HTTPPathInfo}}
```

Comment:
We can simplify the templates even further, if we loop through `api.*` elements, instead of the language-specific ones. The fields used in this level, for example, could be attached directly under the `api.Method.Codec` instead of the `api.HTTPPathInfo`.